models/login_source.go

```diff
@@ -416,6 +416,18 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 		return nil, ErrUserNotExist{0, login, 0}
 	}
 
+	// Update User if exist
+	isExist, err := IsUserExist(0, sr.Username)
+	if err != nil {
+		return nil, err
+	} else if isExist {
+		user.LowerName = strings.ToLower(sr.Username)
```

Comment:
As users are searched in LDAP by username there is no need to update LowerName and Name fields for user. If they will change in LDAP, new user will be created anyway